src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,87 +12,88 @@
     *
     **/
 -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
-           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
+  
+  <reference id="catalogFramework" interface="ddf.catalog.CatalogFramework"/>
+  
+  <reference id="attributeRegistry" interface="ddf.catalog.data.AttributeRegistry"/>
+  
+  <reference-list id="metacardTypes" interface="ddf.catalog.data.MetacardType"/>
+  
+  <reference id="filterBuilder" interface="ddf.catalog.filter.FilterBuilder"/>
+  
+  <reference id="persistentStore" interface="org.codice.ddf.persistence.PersistentStore"/>
```

Comment:
Not sure why the diff came out like this. The only things I added were line 25, 36-38, and line 56.